### PR TITLE
Assorted Trello tasks for SearchLanding and Users

### PR DESCRIPTION
**Pull request by tiffwong**
`SearchLanding` related tasks:
- Remove underline from result labels
- Clean up unused styles and CMR files

`Users` related tasks:
- Add "Users" link to breadcrumb to make it consistent (used to only show up after navigating to the User Permissions tab)
- Remove icon/photo from the breadcrumb
- Disable the save button for username until a change is detected
- Increase the contrast for a disabled email text field 
- Clean up unused styles and CMR files

### CODE REVIEW

File: packages/manager/src/features/Search/ResultRow.tsx [hunk 1]
```diff
@@ -151,82 +72,34 @@ export const ResultRow: React.FC<CombinedProps> = props => {
...
-              [classes.link]: !flags.cmr,
-              [classes.linkCMR]: flags.cmr
-            })}
-            title={result.label}
-          >
-            {result.label}
-          </Link>
-          <Typography variant="body1">{result.data.description}</Typography>
-        </div>
+      <TableCell className={classes.labelCell} parentColumn="Label">
```
Comment (DevDW) [inline packages/manager/src/features/Search/ResultRow.tsx:75]: I remember having some files in PRs earlier on in CMR where the `parentColumn` props were judged to be unnecessary, what do you think about that in these cases here?

**Review by DevDW — APPROVED** [review]
LGTM 👍

File: packages/manager/src/features/Search/ResultRow.tsx (hunk 1 above)
Comment (tiffwong) [reply packages/manager/src/features/Search/ResultRow.tsx:75]: I remember seeing that before, I'll go ahead and remove those!

File: packages/manager/src/features/Users/UserProfile.tsx [hunk 2]
```diff
@@ -134,82 +127,79 @@ class UserProfile extends React.Component<CombinedProps> {
...
+            <Notice success spacingBottom={0}>
+              Email updated successfully
+            </Notice>
+          )}
+          {generalProfileError && (
+            <Notice error text={generalProfileError} spacingBottom={0} />
+          )}
+          <TextField
+            // This should be disabled if this is NOT the current user.
+            disabled={profileUsername !== originalUsername}
```
Comment (Jskobos) [inline packages/manager/src/features/Users/UserProfile.tsx:177]: Seems like this should match the profile input (e.g. also disabled if the user hasn't made a change yet)

**Review by Jskobos — CHANGES_REQUESTED** [review]
Looks good, I pushed up some cleanup, and I'm requesting one change to the email input in UserProfile.

**Review by Jskobos — APPROVED** [review]

**Pull request merged**